src/main/java/com/github/bpiatek/bbghbackend/controller/CrawlersController.java

```diff
@@ -42,4 +42,15 @@
       return ResponseEntity.status(ORDINAL_500_Internal_Server_Error).build();
     }
   }
+
+  @ApiOperation(value = "Stop crawler for 90minut.pl")
+  @ApiResponses(value = {
+      @ApiResponse(code = ORDINAL_202_Accepted, message = "Crawler successfully stopped"),
+  })
+  @PostMapping("/stop/90minutes")
+  ResponseEntity<Void> stopCrawler() {
+    facade.stopCrawler();
+    log.info("Crawler for portal 90minut.pl stopped manually.");
+    return ResponseEntity.accepted().build();
```

Comment:
Dałbym tutaj kod 200 `return ResponseEntity.ok().build();` bo tutaj dajemy znać "juz zatrzymalismy, jest ok!",
w przypadku powyzej bylo 202 (accepted) bo ten kod dotyczy:
"Przyjęto – zapytanie zostało przyjęte do obsłużenia, lecz jego zrealizowanie jeszcze się nie skończyło" a my puszczalismy crawler i dawalismy znac "spoko, uruchomilismy crawler i leci"

Reply:
Jak najbardziej racja.